padrino-admin/lib/padrino-admin/access_control.rb

```diff
@@ -96,7 +104,7 @@ def project_modules(account)
         #    = link_to 'Profile', url(:accounts, :edit, :id => current_account.id)
         #
         def allowed?(account=nil, path=nil)
-          path = "/" if path.blank?
+          path ||= "/"
```

Comment:
This doesn't have the same meaning as checking for `path.blank?` if the caller sends `allowed?(account, "")` this won't behave the same.

should be

```
path = "/" if path.nil? || path.empty?
```